skycoin/skywire-services
# Fix docker build on MacOS

Fixes: #18 

Changes:
- add two command on Makefile, `docker-build` and `docker-build-test`
- update docker_build.sh script for make images on linux/amd64 platform, then we can run on all OSs (tested on macbook m1)

How to test:
- change line 18 of `docker_build.sh` script, and set `registry=<your-dockerhub-account>`
- run making images on MacOS machine process by `make docker-build-test`, stop it after first image created `transport-discovery`
- push `<your-dockerhub-account>/transport-discovery:test` image to your repo
- check dockerhub image description for more details on its os and arch
- also  pull and run image on linux machine with `amd64` arch
